Settings/Refractored.Xam.Settings.iOS/Settings.cs

```diff
@@ -225,6 +225,11 @@ public void Remove(string key)
             }
         }
 
+        private NSUserDefaults GetDefaults()
+        {
+            return string.IsNullOrWhiteSpace(CrossSettings.DefaultsName) ? NSUserDefaults.StandardUserDefaults
+                : new NSUserDefaults(CrossSettings.DefaultsName, NSUserDefaultsType.SuiteName);
```

Comment:
Not sure if we want this to get newed up like this every time you request a setting or not. Maybe a static reference from CrossSettings would be more appropriate?